server/src/http.rs

```diff
@@ -459,6 +459,11 @@ impl<Q: QueryExecutor + 'static> Builder<Q> {
             enable_tenant_as_schema: self.enable_tenant_as_schema,
         };
 
+        info!(
+            "HTTP server starts listening on {}",
+            &self.config.endpoint.to_string()
+        );
```

Comment:
How about printing this after spawning?

Reply:
I guess you won't know which port will be used by the http service, so how about updating this log to:
```suggestion
        info!(
            "HTTP server tries to listen on {}",
            &self.config.endpoint.to_string()
        );
```

And add one more log after it succeeds in binding:
```
        info!(
            "HTTP server listens on {}",
            &self.config.endpoint.to_string()
        );
```

Reply:
The log is verbose, I prefer to only add the first one.

Reply:
I add panic logic when listen failed for grpc and mysql, http server already does this.